x-pack/plugins/aiops/common/api/stream_reducer.ts

```diff
@@ -35,6 +35,9 @@ export function streamReducer(
     return action.reduce(streamReducer, state);
   }
 
+  // eslint-disable-next-line no-console
+  console.log('action', action);
```

Comment:
I'm assuming this console.log is here intentionally... In which case might be good to have a better log prefix message. But this is a super duper nit and is not an issue. Please feel free to merge 😄 .

Reply:
Sorry for not mentioning this, it's just for some temporary cloud testing and will be deleted before merging!

Reply:
Removed in [2bc9fce](https://github.com/elastic/kibana/pull/142047/commits/2bc9fce722be81950c6d38a2c89e4639c77790a4).